packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -106,6 +106,7 @@ export const SELL_SOURCE_FILTER_BY_CHAIN_ID = valueByChainId<SourceFilters>(
             ERC20BridgeSource.CurveV2,
             ERC20BridgeSource.ShibaSwap,
             ERC20BridgeSource.Synapse,
+            ERC20BridgeSource.RadioShack,
```

Comment:
`BUY_SOURCE_FILTER_BY_CHAIN_ID` should be updated too (line 227)

Reply:
Ah! Good catch. Ty, ser!